index.js

```diff
@@ -0,0 +1,97 @@
+import { catsData } from "/data.js"
+
+const catsArray = []
+const emotionsArray = []
```

Comment:
```suggestion
const emotions = []
```

Reply:
no point in defining this variable in the scope of the whole module, you only use it in `getEmotionsArray`

Reply:
Addressed, moved to getEmotionsArray